src/Compilers/Core/Portable/WellKnownTypes.cs

```diff
@@ -252,10 +251,10 @@ internal enum WellKnownType
         CSharp7Sentinel = System_IFormatProvider, // all types that were known before CSharp7 should remain above this sentinel
 
         System_ValueTuple,
-        System_ValueTuple_T1,
```

Comment:
>System_ValueTuple_T1, [](http://example.com/codeflow?start=8&length=21)

I do not expect any changes in this file. Please revert to its original state.

Reply:
Special and well-known types share the same number space: the first well-known type goes after the last special one. Since we added a special type we have to move marker well-known type with code 255 one type below. This causes changes in well-known types/members files

Reply:
> This causes changes in well-known types/members files

Makes sense. Thanks for the explanation